boudicca.base/enricher/src/main/kotlin/base/boudicca/enricher/EnricherConfiguration.kt

```diff
@@ -4,8 +4,11 @@ import io.swagger.v3.oas.annotations.OpenAPIDefinition
 import io.swagger.v3.oas.annotations.info.Info
 import io.swagger.v3.oas.annotations.info.License
 import io.swagger.v3.oas.annotations.servers.Server
+import org.springframework.boot.autoconfigure.AutoConfiguration
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
 import org.springframework.scheduling.annotation.EnableScheduling
 
 @OpenAPIDefinition(
```

Comment:
Why is the OpenApiDefinition part of the configuration? Shouldn't that be in the enricher-api module provided as interface and then used in the controller?

Reply:
i cannot remember exactly why i did it this way but i only know that other ways did not work... maybe they work now but i am not motivated to spend more time on this